Added documentation on how to troubleshoot the case of scaping chars in CLI

juanmatias opened the pull request:
## What?
* Added documentation on how to escape chars using the command line for resource arrays.

## Why?
* It is not a common case, but it worth to be documented.

## References
N/A

## Important
Please include a screenshot of the pages that you modified to help reviewers. If you are using Chrome, you can follow the instructions below to easily take a full screenshot of the entire page:
1. Right click the page and open the inspector.
2. In the top left of Chrome Inspector you'll see a little phone icon whose tooltip shows "Toggle device emulation".
3. Hit that and a new black bar will appear on the top of the webpage.
4. In the black bar you can pick the dimensions of the screen you want to stimulate -- Since the documentation uses a responsive layout, you'll probably want to set a wide enough value on the width field, e.g. 1024x768, then the height is not as important.
5. In top right of black bar is three dots, in that menu you can do a full page screenshot.

![image](https://github.com/user-attachments/assets/5a82b4f3-611d-4bf6-ad01-ed67254d5c7f)

exequielrafaela commented on docs/user-guide/troubleshooting/general.md:
````diff
@@ -102,3 +102,20 @@ In order to fix this, just configure git globally:
+To do this escape the target string like this:
+
+```shell
+leverage tf import 'aws_organizations_account.accounts[\"accountalias\"]' 99999999999
````
This is already documented here https://leverage.binbash.co/user-guide/leverage-cli/reference/terraform/#import

exequielrafaela reviewed (changes requested):
@juanmatias Let's update this PR based on our internal discussion here: https://binbashar.slack.com/archives/GG0PJ78J3/p1730144578139259?thread_ts=1730142394.792759&cid=GG0PJ78J3

When ready let us know and we can approve it and merge it.
Thanks for your collaboration!

juanmatias commented:
@exequielrafaela @diego-ojeda-binbash 

Added notes on `noglob` for `import`.

Added notes for `-target` on `plan`, `apply` and `destroy`. Didn´t add notes on noglob because it didn't work on this case for me (neither on `bash` or `zsh`).
